model/src/main/java/org/projectnessie/api/params/CommitLogParams.java

```diff
@@ -15,11 +15,6 @@
  */
 package org.projectnessie.api.params;
```

Comment:
I would prefer to keep it as it is for now until we have more `Param` classes, where it gets clearer how the Base class could look. Even though the 3 fields are identical, only two have the same parameter description. Not sure it's worth adding inheritance for 2 fields atm.

Reply:
I agree w/ @nastra. I think if anythign we would get rid of these objects and return to 3 parameters in the actual jax-rs method. We only intruduced these objects to make it easy/clean to add multiple filters, which we have now folded into CEL. 

Just food for thought, not needed for this PR

Reply:
indeed it would probably make more sense to move those 3 params to the actual method again

Reply:
opened https://github.com/projectnessie/nessie/issues/1479 to address that outside of this PR as the required changes would otherwise just pollute everything